sanitytests/axku040/resources/Makefile

```diff
@@ -0,0 +1,8 @@
+RISCV ?= riscv-unknown-elf-
+OBJCOPY ?= $(RISCV)objcopy
+GCC ?= $(RISCV)gcc
+bootrom.bin: bootrom.elf
+	$(OBJCOPY) -O binary $< $@
+
+bootrom.elf: bootrom.c i2c.h linker.ld
+	$(GCC) -march=rv64gc -mabi=lp64 -Os -static -nostdlib -mno-relax -Wl,--no-gc-sections -T linker.ld -o $@ bootrom.c
```

Comment:
It is probably possible but I don't really want to do so because target triplet is in the name of toolchain and not necessarily same across different distributions (like `riscv-unknown-elf` vs `riscv64-none-elf` etc)

Probably I would like just rewrite `bootrom.c` to be clang compatible and use the same build process as `vcu118`.